src/azure-cli/azure/cli/command_modules/managedservices/_help.py

```diff
@@ -0,0 +1,174 @@
+        - name: Gets the registration definition given its identifier and subscription id.
+          text: az managedservices definition show --definition af8772a0-fd9c-4ddc-8ad0-7d4b3913d7dd --subscription 39033314-9b39-4c7b-84fd-0e26e55f15dc
+        - name: Gets the registration definition given its fully qualified resource id.
+          text: az managedservices definition show --definition /subscriptions/39033314-9b39-4c7b-84fd-0e26e55f15dc/providers/Microsoft.ManagedServices/registrationDefinitions/1d693e4f-9e79-433a-b3a2-afce1f8b61ec
+"""
+
+helps['managedservices definition delete'] = """
+    type: command
+    short-summary: Deletes a registration definition.
+    long-summary: Deletes a registration definition when given its identifier or the fully qualified resource id. When resource id is provided, the subscription paramter is ignored.
+    parameters:
+        - name: --definition
+          short-summary: |
+             The identifier (guid) or the fully qualified resource id of the registration definition. When resource id is used, subscription id and resource group parameters are ignored.
+    examples:
+        - name: Deletes the registration definition given its identifier under the default subscription scope.
+          text: az managedservices definition delete --definition af8772a0-fd9c-4ddc-8ad0-7d4b3913d7dd
+        - name: Deletes the registration definition given its identifier and subscription id.
+          text: az managedservices definition delete --definition af8772a0-fd9c-4ddc-8ad0-7d4b3913d7dd --subscription 39033314-9b39-4c7b-84fd-0e26e55f15dc
+        - name: Deletes the registration definition given its fully qualified resource id.
+          text: az managedservices definition delete --definition /subscriptions/39033314-9b39-4c7b-84fd-0e26e55f15dc/providers/Microsoft.ManagedServices/registrationDefinitions/1d693e4f-9e79-433a-b3a2-afce1f8b61ec
+"""
+
+helps['managedservices definition list'] = """
+    type: command
+    short-summary: List all the registration definitions.
+    long-summary: Lists all the registration definitions under the default scope or under the subscription provided.
+    examples:
+        - name: Lists all the registration definitions under the default subscription scope.
+          text: az managedservices definition list
+        - name: Lists all the registration definitions under the given subscription scope.
+          text: az managedservices definition list --subscription 0c3e9687-b461-4615-b6e4-74d54998d6e4
+"""
+
+helps['managedservices definition assignment create'] = """
+    type: command
+    short-summary: Creates a new registration assignment.
+    long-summary: Creates a new registration assignment when given the fully qualified resource id of the registration definition.
+    parameters:
+        - name: --definition-id
+          short-summary: |
+             The fully qualified resource id of the registration definition.
```

Comment:
If this is an arm resource, it should be `--definition` and should follow the logic that `--definition` does in other commands.